07-wt-ed-ds-pipeline.Rmd

```diff
@@ -545,11 +557,14 @@ survey_responses <-
     )
   ) %>%
   group_by(student_id, measure) %>%
-  summarise(# Here's where we compute the mean of the responses
+  # Here's where we compute the mean of the responses
+  summarise(
     # Mean response for each measure
-    mean_response = mean(response, na.rm = TRUE)) %>%
+    mean_response = mean(response, na.rm = TRUE)
+    ) %>%
   filter(!is.na(mean_response)) %>%
-  spread(measure, mean_response)
+  pivot_wider(names_from = measure, 
```

Comment:
thanks for adding this - and also adding the names for the arguments to `pivot_wider()` (helpful to me to learn)